src/Onboarding/EulaModal.tsx

```diff
@@ -77,14 +75,22 @@ const Checkbox: FunctionComponent<CheckboxProps> = ({
   )
 }
 
-const DEFAULT_EULA_URL = "about:blank"
-
-type AvailableLocale = "en" | "es_PR" | "ht"
+const Spinner = () => {
+  return (
+    <ActivityIndicator
+      size={"large"}
+      color={Colors.darkGray}
+      style={styles.activityIndicator}
+      testID={"loading-indicator"}
+    />
+  )
+}
 
-const EULA_FILES: Record<AvailableLocale, string> = {
-  ["en"]: en,
-  ["es_PR"]: es_PR,
-  ["ht"]: ht,
+const eulaPicker = (selectedLocale: string): string => {
+  if (selectedLocale) {
+    return `${EULA_URL}/${selectedLocale}`
+  }
+  return `${EULA_URL}/en`
```

Comment:
```suggestion
const eulaPicker = (selectedLocale? = "en"): string => {
    return `${EULA_URL}/${selectedLocale}`
}
```

Could this work? Maybe simplifies the method?